specification/storagecache/resource-manager/Microsoft.StorageCache/preview/2020-08-13-aml-preview/examples/amlFilesystems_CreateOrUpdate.json

```diff
@@ -0,0 +1,156 @@
+{
+  "parameters": {
+    "resourceGroupName": "scgroup",
+    "amlFilesystemName": "fs1",
+    "subscriptionId": "00000000-0000-0000-0000-000000000000",
+    "api-version": "2020-07-23-preview",
+    "amlFilesystem": {
+      "tags": {
+        "Dept": "ContosoAds"
+      },
+      "location": "westus",
+      "properties": {
+        "storageCapacityTiB": 3072,
+        "zones" : ["1", "2"],
+        "subnet": "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/scgroup/providers/Microsoft.Network/virtualNetworks/scvnet/subnets/sub1",
+        "MSGAddress" : "10.0.0.4",
```

Comment:
MSGAddress is read-only; callers may not set/change it

Reply:
Fixed.